x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -94,7 +94,12 @@ export const AddModelFlyout: FC<AddModelFlyoutProps> = ({ onClose, onSubmit, mod
   }, [selectedTabId, tabs]);
 
   return (
-    <EuiFlyout ownFocus onClose={onClose} aria-labelledby={'addTrainedModelFlyout'}>
+    <EuiFlyout
+      ownFocus
+      onClose={onClose}
+      aria-labelledby={'addTrainedModelFlyout'}
+      data-test-subj={'addTrainedModelFlyout'}
```

Comment:
we usually prefix test subject with `ml`
```suggestion
      data-test-subj={'mlAddTrainedModelFlyout'}
```